waterlog/src/Components/DailyCostsReports.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Line, defaults } from 'react-chartjs-2';
+
+const DailyCostsReports = (props) => { 
+  var labelX = props.props.dataPoints.map(a => (new Date(a.x).getHours() + ":00"));
+  var dataY = props.props.dataPoints.map(a => a.y);
+  var today = new Date(Date.now());
+  var data = {
+    labels: labelX,
+    datasets: [
+      {
+        label: 'Rands',
+        data: dataY,
+        fill: true,
+        borderColor: 'red',
+        backgroundColor: 'rgba(255,0,0,0.4)'
+      }
+    ]
+  }
+  var options = {
+    scales: {
+      xAxes: [{
+        scaleLabel: {
+          display: true,
+          labelString: 'Hours'
+        },
+        ticks: {
+          major: {
+            fontStyle: 'bold',
+            fontColor: '#FF0000'
```

Comment:
line 16 is rgba and this is hex. 
check what the project standard is and keep it the same

Reply:
Thing is the rgba is the only way to set a background opacity.

Reply:
Changed to rgb.